### PR TITLE
Fixing check style and find bugs issues

### PR DESCRIPTION
**Pull request by manuelortiz28**
findbugs_issues

### CODE REVIEW

File: app/src/main/java/com/visiontech/yummysmile/ui/fragments/CreateMealFragment.java [hunk 1]
```diff
@@ -252,10 +251,10 @@ private void selectFromGalleryResult(Intent data) {
                     setFinalView(renderImageHelper.getRenderedBitmap());
                     pictureToUpload = renderImageHelper.getImageToUpload();
                 }
+                cursor.close();
```
Comment (htorrex) [inline app/src/main/java/com/visiontech/yummysmile/ui/fragments/CreateMealFragment.java]: +1

File: app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java [hunk 2]
```diff
@@ -92,16 +96,14 @@ public boolean resizeFileBitMap(String filePath) {
...
         renderedBitmap.compress(Bitmap.CompressFormat.JPEG, QUALITY_PICTURE, bos);
 
-        try {
-            imageToUpload = new File(filePath);
-            FileOutputStream fo = new FileOutputStream(imageToUpload);
+        imageToUpload = new File(filePath);
+
+        try (FileOutputStream fo = new FileOutputStream(imageToUpload)) {
             fo.write(bos.toByteArray());
-            fo.close();
```
Comment (htorrex) [inline app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java]: question: Why are we not closing the stream?.

File: app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java [hunk 3]
```diff
@@ -70,6 +72,7 @@ public File createImageFile() throws IOException {
     /**
      * Method that resize the image before showing in preview.
      */
+    @TargetApi(Build.VERSION_CODES.KITKAT)
```
Comment (htorrex) [inline app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java]: Why are we forcing to use kitkat? probably if there's something that we will have to validate in M or below that we should fix in code more than force lint with the annotation.

File: app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java [hunk 4]
```diff
@@ -92,16 +96,14 @@ public boolean resizeFileBitMap(String filePath) {
         ByteArrayOutputStream bos = new ByteArrayOutputStream();
         renderedBitmap.compress(Bitmap.CompressFormat.JPEG, QUALITY_PICTURE, bos);
 
-        try {
-            imageToUpload = new File(filePath);
-            FileOutputStream fo = new FileOutputStream(imageToUpload);
+        imageToUpload = new File(filePath);
+
+        try (FileOutputStream fo = new FileOutputStream(imageToUpload)) {
```
Comment (htorrex) [inline app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java]: Could explain what is the difference/advantage to put this sentence please?

**Comment by htorrex** [comment]
Thanks to run the check and pdm i forgot when I upload the code. i have some questions though.

File: app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java (hunk 4 above)
Comment (manuelortiz28) [inline app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java]: With this sentence, you don't need to close this resource. For more reference https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

File: app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java (hunk 2 above)
Comment (manuelortiz28) [inline app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java]: The stream is auto closable using the try (...) sentence

File: app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java (hunk 3 above)
Comment (manuelortiz28) [inline app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java]: Autoclosable only is supported starting with kitkat

**Comment by manuelortiz28** [comment]
@htorrex I pushed more changes

File: app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java (hunk 4 above)
Comment (htorrex) [inline app/src/main/java/com/visiontech/yummysmile/util/RenderImageHelper.java]: 👍  nice

**Comment by htorrex** [comment]
LG

**Pull request merged**